languages/python/exercises/concept/enums/.meta/example.py

```diff
@@ -0,0 +1,40 @@
+from enum import Enum
+
+class LogLevel(Enum):
+    Trace = 'TRC'
+    Debug = 'DBG'
+    Info = 'INF'
+    Warning = 'WRN'
+    Warn = 'WRN'
+    Error = 'ERR'
+    Fatal = 'FTL'
+    Unknown = 'UKN'
+
+class LogLevelInt(Enum):
+    Trace = 0
+    Debug = 1
+    Info = 4
+    Warning = 5
+    Warn = 5
+    Error = 6
+    Fatal = 7
+    Unknown = 42
+
+def parse_log_level(message):
+    str_split = message.split(":")
+    log_values = [level.value for level in LogLevel]
+    if str_split[0][1:-1] in log_values:
+        return LogLevel(str_split[0][1:-1])
+    return LogLevel.Unknown
```

Comment:
We shouldn't use syntax that we haven't either _introduced_ or _required as a prerequisite_.  Do we want to add `indexing`, `slicing` `string methods`, `string formatting`, and `list comprehensions` as requirements for this exercise?